README.md

```diff
@@ -1,3 +1,5 @@
+![GPU Integration Test](https://github.com/pytorch/torchtitan/actions/workflows/unit_test_4gpu.yaml/badge.svg)
```

Comment:
In this example, they link this badge to the actual test actions. That could be convenient to identify where test breaks.

https://github.com/pytorch/torchtune/blob/main/README.md?plain=1#L2

Reply:
Good idea, added